content/en/docs/releases/kubeflow-1.8.md

```diff
@@ -0,0 +1,278 @@
+
+<div class="table-responsive">
+<table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Maintainers</th>
+        <th>Component Name</th>
+        <th>Version</th>
+      </tr>
+    </thead>
+  <tbody>
+      <!-- ======================= -->
+      <!-- AutoML Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">AutoML Working Group</td>
+        <td>Katib</td>
+        <td>
+          <a href="https://github.com/kubeflow/katib/releases/tag/v0.16.0">v0.16.0</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Notebooks Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="9" class="align-middle">Notebooks Working Group</td>
+        <td>Admission Webhook (PodDefaults)</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/admission-webhook">v1.8.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Central Dashboard</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/centraldashboard">v1.8.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Jupyter Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/crud-web-apps/jupyter">v1.8.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Kubeflow Access Management API</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/access-management">v1.8.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Notebook Controller</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/notebook-controller">v1.8.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Profile Controller</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/profile-controller">v1.8.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Tensorboard Controller</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/notebook-controller">v1.8.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Tensorboard Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/crud-web-apps/volumes">v1.8.0</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Volumes Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/crud-web-apps/tensorboards">v1.8.0</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Pipelines Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="2" class="align-middle">Pipelines Working Group</td>
+        <td>Kubeflow Pipelines</td>
+        <td>
+          <a href="https://github.com/kubeflow/pipelines/releases/tag/2.0.3">2.0.3</a>
```

Comment:
A minor nit so that it's formatted consistently across the table and past versions:

```suggestion
          <a href="https://github.com/kubeflow/pipelines/releases/tag/2.0.3">v2.0.3</a>
```

Reply:
Noticed this also, but realized that this way it's consistent with the tag that is available.